ISSUE_TEMPLATE/custom_template.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
placeholder?

Reply:
This allows people to make an issue without any templating, serves any possible edge case outside of your feature request and bug templates